crates/resource-resolver/src/starting_binary.rs

```diff
@@ -0,0 +1,77 @@
+// Copyright 2019-2023 Tauri Programme within The Commons Conservancy
```

Comment:
this is now a replica of `updater/src/starting_binary.rs`, we should pull it into a common crate so we don't have duplicate code on our hand. Probably `cargo-packager-utils`